.azure-pipelines/setup.yml

```diff
@@ -10,7 +10,7 @@ steps:
     conda install -c conda-forge --yes mamba
   displayName: Installing mamba in base env
 - bash: |
-    conda create -n foo -q --yes --quiet -c conda-forge -c bioconda snakemake=6.2.1 fuzzywuzzy mock sphinx sphinx-argparse sphinx_rtd_theme flake8 coreutils python=3.9
+    conda create -n foo -q --yes --quiet -c conda-forge -c bioconda snakemake=6.2.1 fuzzywuzzy mock sphinx sphinx-argparse sphinx_rtd_theme jinja2=3.0.3 flake8 coreutils python=3.9
```

Comment:
Is this for rendering the docs?

Reply:
thanks for spotting that one! I have to fix the sphinx version here, too, and remove jinja2 downgrade.

Reply:
actually, since we're planning to drop azure pipelines right after this bugfix release, I wouldn't mind to leave this unfixed. I can see that the docs are building with the `docs/environment.yml` and `docs/requirements.txt` for the develop branch on the read the docs server, which is more important to me. Would you agree?